mozci/task.py

```diff
@@ -614,6 +647,31 @@ def failing_tasks(self):
             )
         ]
 
+    def is_confirmed_failure(self) -> Optional[bool]:
+        # match labels to -cf labels, and if ran, return True || False
+        # if not run -cf task, return None
+        # return True = intermittent
+        # return False = regression
```

Comment:
I'd do the opposite `True` means it is a confirmed failure (so regression), `False` means it is not (so intermittent).
This would match better the name of the function (and its usage in mozci/push.py).